controller/src/main/java/io/pravega/controller/store/stream/PravegaTablesStream.java

```diff
@@ -812,6 +812,44 @@ private String getWritersTableName(String id) {
         });
     }
 
+    @Override
+    public CompletableFuture<List<UUID>> listCompletedTransactions(OperationContext context) {
+        Preconditions.checkNotNull(context, "operation context cannot be null");
+
+        return getAllBatchesForCompletedTxns(context)
```

Comment:
We may need to change this if we want to use continuation token and always consider only the last batch

Reply:
As discussed we should be picking all txns from `transactionsInBatch-0` table.

Reply:
now picking all txns from` transactionsInBatch-0` table.